### ISSUE TITLE
Add/update headings in the tab sections on the Collection/Component pages

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#837]
Part of #820

Currently, the Overview and Contents tab have hidden `<h2>`s that use deprecated wording. The other tabs have no headings.

- [ ] Each tab panel should have an `<h2 class="sr-only">` as the first item in the panel (except Overview, which is fine where it is). 

- [ ] Each heading should use the same text as its parent tab, in a way that ensures they will remain in sync as the visible tab label is changed or translated.

- [ ] When the tab panel is empty (e.g. the tab is present but says "No online content"), the `<h2>` should not be present.
